draft_python/trials.py

```diff
@@ -0,0 +1,96 @@
+import argparse
+import contextlib
+import os
+from mtg_draft_ai.controller import *
+from mtg_draft_ai.api import *
+from mtg_draft_ai.brains import *
+from mtg_draft_ai import draftlog, deckbuild, display
+
+
+def main():
+    parser = argparse.ArgumentParser(description='Run N full drafts + deckbuilds for each, with all logs saved.')
+    parser.add_argument('n', type=int, help='Number of drafts to run')
+    parser.add_argument('--card-data', type=str, help='Card data TOML file', default='cube_81183_tag_data.toml')
+    parser.add_argument('-d', '--dir', type=str, help='Output directory for files', default='drafts')
+
+    args = parser.parse_args()
+
+    cube_list = read_cube_toml(args.card_data)
+    draft_info = DraftInfo(card_list=cube_list, num_drafters=6, num_phases=3, cards_per_pack=15)
+    drafter_factory = GreedySynergyPicker.factory(cube_list)
+
+    edge_counts = []
+    for i in range(0, args.n):
+        edge_counts += run_trial(name=i, output_dir=args.dir, draft_info=draft_info, drafter_factory=drafter_factory,
+                                 deckbuild_fn=deckbuild.best_two_color_synergy_build)
+
+    result = sum(edge_counts) / len(edge_counts)
+    print('Avg. # of edges: {}'.format(result))
+
+
+def run_trial(name, output_dir, draft_info, drafter_factory, deckbuild_fn):
+    draft_log_file = os.path.join(output_dir, 'draft-log_{}.txt'.format(name))
+    draft_html_file = os.path.join(output_dir, 'draft_{}.html'.format(name))
+    draft_debug_file = os.path.join(output_dir, 'draft-debug_{}.txt'.format(name))
+    build_html_file = os.path.join(output_dir, 'build_{}.html'.format(name))
+    build_debug_file = os.path.join(output_dir, 'build-debug_{}.txt'.format(name))
+
+    drafters = [Drafter(drafter_factory.create(), draft_info) for _ in range(0, draft_info.num_drafters)]
+
+    # Run draft, redirecting output to debug file
+    with open(draft_debug_file, 'w') as f:
+        with contextlib.redirect_stdout(f):
```

Comment:
At some point you may want to consider moving to a proper logging library instead of using prints and redirecting stdout.

Reply:
Sounds good. I'll try that in the next PR.